### PR TITLE
Use minimal travis container since we install go ourselves.

### PR DESCRIPTION
**Pull request by winder**
## Summary
Since we manage go ourselves I want to see if our build works without running the travis go setup step.

### CODE REVIEW

**Comment by tsachiherman** [comment]
I think that there are some differences in the checkout location on travis go builds. But I would be happy to align with with a minimal travis build if that works.

File: scripts/travis/codegen_verification.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ function runGoFmt() {
 }
 
 function runGoLint() {
-    warningCount=$("$GOPATH"/bin/golint $(GO111MODULE=off go list ./... | grep -v /vendor/ | grep -v /test/e2e-go/) | wc -l | tr -d ' ')
+    warningCount=$("$GOPATH"/bin/golint $(GO111MODULE=on go list ./... | grep -v /vendor/ | grep -v /test/e2e-go/) | wc -l | tr -d ' ')
```
Comment (tsachiherman) [inline scripts/travis/codegen_verification.sh:45]: At this point, can't we just omit the `GO111MODULE=on` ?

Comment (winder) [reply scripts/travis/codegen_verification.sh]: I'll give it a try, it should work since we export it that way.

**Review by tsachiherman — APPROVED** [review]
looks good.

**Pull request merged**